src/fontFace.ts

```diff
@@ -6,6 +6,11 @@ export const generateFontFace = (
   options: TypesettingOptions = { }
 ): string => {
   const { family, variants } = typesettings;
+
+  if (!variants) {
+    throw Error('Missing variants in your typesettings');
+  }
+
```

Comment:
I don't mind throwing here, because there is a change this lib gets utilized w/ vanilla JS, but for TS, we should make this a required prop, right? https://github.com/buames/typesettings-js/blob/a07effbb43c8734f939692e9fb4f77fe49fa2ba4/src/types.ts#L44

Reply:
Also, wondering if you just introduced this as a type guard to shut up TS. You could remove if you make the prop required. Up to you...

Reply:
Making a type guard ... but are you saying to throw or not to throw?

Reply:
If you make `Typesettings.variants` required, you don't have to add the typegaurd and throw at runtime:

```
export interface Typesettings {
  family: FontFamilyProperty;
  fallbacks?: FontFamilyProperty[];
  variants: FontVariant[];
}
```

Reply:
Yup got it. Also made a few others required in the last commit as well allowing for the TypesettingOptions to be set by the project.

Reply:
Ok. You shouldn't need to even handle this use case when it isn't an array then, because the type requires it to be an array. I think you can scrap this whole block:

```
  if (!Array.isArray(variants)) {
    return '';
  }
```

Reply:
removed